### ISSUE TITLE
Copy/transfer from old app to a new

### ISSUE THREAD

**Issue by therethere** [ruchit47/onesignal-user-transfer#1]
Hi there, 

Is this script still working?

If we import the users from OLD onesignal app to NEW onesignal app how can we send pushs notifications to the users that have the app with the OLD onesignal app ID on Android/IOS?

many thanks!

**Comment by ruchit47** [comment]
Yes, It still works.   I just tested, You can still send msg message to your old one signal AppID. Basically, when you copy, you can send notifications to both new and old app ids.  Unless the GCM tokens are not updated.

**Comment by AbreuY** [comment]
Hi @ruchit47 this script still work?

**Comment by AbreuY** [comment]
@ruchit47 How i can execute this script? From web? or using any other software?
